packages/sdk/src/Vouchers.ts

```diff
@@ -89,4 +89,16 @@ export class Vouchers {
 	public releaseValidationSession(code: string, sessionKey: string) {
 		return this.client.delete(`/vouchers/${encode(code)}/sessions/${encode(sessionKey)}`)
 	}
+	/**
+	 * @see https://docs.voucherify.io/reference/list-gift-card-transactions
+	 */
+	public listGiftCardTransactions(code: string, query?: T.ListGiftCardTransactionsQuery) {
+		return this.client.get<T.ListGiftCardTransactionsResponse>(`/vouchers/${encode(code)}/transactions`, query)
+	}
+	/**
+	 * @see https://docs.voucherify.io/reference/export-gift-card-transactions
+	 */
+	public exportGiftCardTransactions(code: string, body: T.ExportGiftCardTransactionsBody) {
+		return this.client.post<T.ExportGiftCardTransactionsResponse>(`/vouchers/${encode(code)}/transactions/export`, body)
```

Comment:
just to be sure?
Is there a case when you will get
```
{
...
status: DONE
result: string
}
```
here?

Reply:
it looks like it's an endpoint dedicated only to schedule the export and it always returns "status": "SCHEDULED", need to check this